tests/tests.js

```diff
@@ -170,6 +191,41 @@ var runTests = function runTests(arrayFrom) {
 		t.end();
 	});
 
+	test('works with iterable objects', function (t) {
+		(function () {
+			t.deepEqual(arrayFrom(arguments), [1, 2, 3]);
+		}(1, 2, 3));
+
+		test('works with Array.prototype.values', { 'skip': !Array.prototype.values }, function (te) {
+			te.deepEqual(arrayFrom([1, 2, 3].values()), [1, 2, 3]);
+		});
+
+		test('works with Map objects', { 'skip': 'Map' in global && 'values' in Map.prototype }, function (te) {
```

Comment:
this will skip the test when `Map` *is* present - i think you need to invert the condition.